packages/cap-config-writer/src/cap-config/index.ts

```diff
@@ -42,21 +44,24 @@ export async function enableCdsUi5Plugin(basePath: string, fs?: Editor): Promise
  *
  * @param basePath - root path of the CAP project, where package.json is located
  * @param [fs] - optional: the memfs editor instance
+ * @param {CdsVersionInfo} [cdsVersionInfo] - Optional. Information about the CDS version which is extracted during the prompting phase of app generation.
```

Comment:
Could you please explain what happens when this is passed in the doc?

Reply:
Thanks Iain,  I added some more explanation as for what we use CdsVersionInfo for?

Reply:
Still a little confusing 
```suggestion
 * @param {CdsVersionInfo} [cdsVersionInfo] - If provided will be used instead of parsing the package.json file to determine the cds version.
```